### PR TITLE
(BIDS-1059) Invert notification thresholds

### PR DESCRIPTION
**Pull request by Eisei24**
### <samp>🤖 Generated by Copilot at 754ec45</samp>

Fix disk space notifications in `notificationsCenter.js`. Invert the event threshold for the "monitoring_hdd_almostfull" event to match the disk usage percentage.

### CODE REVIEW

File: static/js/notificationsCenter.js [hunk 1]
```diff
@@ -148,10 +148,15 @@ function loadMonitoringData(data) {
   }
 
   for (let i = 0; i < data.length; i++) {
+    var eventThreshold = data[i].EventThreshold
+    if (data[i].EventName === "monitoring_hdd_almostfull") {
+      eventThreshold = 1 - eventThreshold
+    }
```
Comment (D13ce) [inline static/js/notificationsCenter.js:154]: **Minor:** Please add a comment to this block as this is very weird :)

**Review by D13ce — COMMENTED** [review]
utACK, but I left a suggestion. ptal

File: static/js/notificationsCenter.js (hunk 1 above)
Comment (Eisei24) [reply static/js/notificationsCenter.js:155]: Done.

**Review by D13ce — APPROVED** [review]
utACK

**Pull request merged**